apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -84,7 +84,7 @@ const ProgressBubbleSet = React.createClass({
                 <ProgressPill
                   url={level.url}
                   status={level.status}
-                  text={i18n.unpluggedActivity()}
+                  text={level.isGoBeyond ? i18n.goBeyond() : i18n.unpluggedActivity()}
```

Comment:
Yep, I've set it up that way so it follows existing unplugged behaviors (skips over during progression).  It would've been cleaner to create a common subtype that both, but since they share most functionality I think I'm ok with this for now.